FluentUI/src/main/res/values/dimens.xml

```diff
@@ -21,6 +21,20 @@
     <dimen name="fluentui_avatar_size_xlarge">52dp</dimen>
     <dimen name="fluentui_avatar_size_xxlarge">64dp</dimen>
     <dimen name="fluentui_avatar_square_corner_radius">@dimen/fluentui_corner_radius_4</dimen>
+    <dimen name="fluentui_avatar_border_size">4dp</dimen>
+    <dimen name="fluentui_avatar_border_size_xxlarge">8dp</dimen>
+    <dimen name="fluentui_avatar_pile_space_xsmall">4dp</dimen>
```

Comment:
question : border needs to be customized ?

Reply:
Nope,  as per the docs and discussion with Designers border. should be of fixed size for different variants i.e small, XL, L, XXL